app/app.go

```diff
@@ -87,13 +89,14 @@ import (
 	"github.com/tendermint/spm/openapiconsole"
 
 	"github.com/ChihuahuaChain/chihuahua/docs"
-
 	// this line is used by starport scaffolding # stargate/app/moduleImport
 )
 
 const (
 	AccountAddressPrefix = "chihuahua"
 	Name                 = "chihuahua"
+	upgradeName          = ""
```

Comment:
We left this blank, but you should name this constant to whatever you want to name this upgrade. Juno had this named "moneta-alpha" since that was the name of their upgrade. One idea we had was "The Spirit of the Wolf"